ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -1,2298 +1,2315 @@
+    #     self.assertCountEqual(order_data['contacts'], contact_names_list)
+    #     self.info('assert that department updated')
+    #     self.assertEqual([department], order_data['suborders'][0]['departments'])
+    #
+    # def test064_edit_department_of_order_with_multiple_contacts(self):
+    #     """
+    #     In case I select multiple contacts the departments should be updated according to that
+    #
+    #     LIMS-5705 'edit mode'
+    #     """
+    #     self.info("create order with multiple contacts")
+    #     response, payload = self.orders_api.create_order_with_multiple_contacts()
+    #     self.assertEqual(response['status'], 1, response)
+    #     contact_names_list = [contact['text'] for contact in payload[0]['contact']]
+    #     self.info('selected contacts are {}'.format(contact_names_list))
+    #     departments_list_with_contacts = self.contacts_api.get_department_contact_list(contact_names_list)
+    #     self.info('department contacts list {}'.format(departments_list_with_contacts))
+    #     self.info('open edit page of order no {}'.format(payload[0]['orderNo']))
+    #     self.orders_page.get_order_edit_page_by_id(response['order']['mainOrderId'])
+    #     self.order_page.sleep_tiny()
+    #     suggested_department_list, departments_only_list = \
+    #         self.order_page.get_department_suggestion_lists(open_suborder_table=True, contacts=contact_names_list)
+    #     self.info('suggested department list {}'.format(suggested_department_list))
+    #     self.info('and it should be {}'.format(departments_list_with_contacts))
+    #     index = 0
+    #     for item in suggested_department_list:
+    #         for element in departments_list_with_contacts:
+    #             if item['contact'] == element['contact']:
+    #                 self.assertCountEqual(item['departments'], element['departments'])
+    #                 index = index + 1
+    #     self.assertEqual(index, len(contact_names_list))
+    #
+    #     department = random.choice(departments_only_list)
+    #     self.info('set department to {}'.format(department))
+    #     self.order_page.set_departments(department)
+    #     self.order_page.save_and_wait(save_btn='order:save')
+    #     suborder_data = self.order_page.get_suborder_data()
+    #     self.assertEqual([department], suborder_data['suborders'][0]['departments'])
+    #
+    # def test065_download_suborder_sheet_for_single_order(self):
+    #     """
+    #     Export order child table
+    #
+    #     LIMS-8085- single order case
+    #     """
+    #     self.info('select random order')
+    #     random_row = self.orders_page.get_random_table_row(table_element='general:table')
+    #     self.orders_page.click_check_box(source=random_row)
+    #     random_row_data = self.base_selenium.get_row_cells_dict_related_to_header(random_row)
+    #     self.orders_page.open_child_table(source=random_row)
+    #     child_table_data = self.order_page.get_table_data()
+ 
```

Comment:
In this test case, you need to open edit page using GUI: by pressing on edit icon in order row